client/lib/user/shared-utils.js

```diff
@@ -75,8 +74,8 @@ export default {
 	},
 
 	getComputedAttributes: function( attributes ) {
-		var language = getLanguage( attributes.language ),
-			primayBlogUrl = attributes.primary_blog_url || '';
+		const language = getLanguage( attributes.language );
+		const primayBlogUrl = attributes.primary_blog_url || '';
```

Comment:
Good occasion to fix the typo here (primary/primay)

Reply:
👍